Payments/Overview/03_Processes.md

```diff
@@ -0,0 +1,17 @@
+# Processes overview
+
+- [Payments documentation](#payments-documentation)  
+- [Payments in the order process](#payments-in-the-order-process)
+
+## Payments documentation
+The following scheme shows the processes and chapters of the *Payments* module documentation and their link to other modules:
+
+![Payments processes](../../Assets/Screenshots/Payments/Processes/PaymentsProcesses.png "[Payments processes]")
+
+
+
+## Payments in the order process
+
+The whole payments steps are part of the order workflow and, except the confirmation menu entry, you can only monitor the steps for errors occurred during processing the order workflow or for communication errors occurred during transferring the data between the payment service provider and Actindo. As these processes are invisible, the following scheme shows the single payments steps in the order workflow.
```

Comment:
```suggestion
The whole payments steps are part of the order workflow and, except for the *Confirmation* menu entry, you can only monitor the steps for errors occurred while processing the order workflow or for communication errors occurred while transferring the data between the payment service provider and Actindo. As these processes are invisible, the following scheme shows the single payments steps in the order workflow.
```
Ich bin hin- und hergerrisen mit "invisible"... Die passieren in Hinterground bei Actindo, automatisiert, über den Treiber, in externen Systemen (Shop, PSP...), aber man kann sie noch nachvollziehen und teilweise auch sehen, daher finde ich schwer, sie invisible zu nennen.

Reply:
During ist besseres Englisch, Invisible gelöscht